src/waypoint.jsx

```diff
@@ -333,13 +352,18 @@ export default class Waypoint extends React.Component {
    * @return {Object}
    */
   render() {
+    if (this.props.children) {
+      return <div ref={this.refElement}>{this.props.children}</div>;
```

Comment:
I haven't used `refToMeasure` before, and I wasn't able to figure out what that would look like from reading those links. Could someone type up a very quick example showing what that would look like for this lib?

The primary benefit of using `React.Children.only` that I see is allowing users to customize any number of props of the wrapping node. With that said, I think it's likely that most developers will just want to apply styling to the wrapper, rather than any other props, so I'm not sure how much that benefit will matter to folks.

I understand, and have experienced, many of the problems described in the "Don’t pass CSS classes between components" post, yet I disagree with the conclusion that categorically denying use of class name / style props is the best solution. I don't want to get too much into that here (although it'd be a great conversation, I'm sure!), but I figured I'd mention that I'd be fine just going the more traditional route and using class name/style props.

Anyway, I'm likely also alright with the `refToMeasure` prop as long as it's not too complex nor confusing for the consumers of this lib.